zenoh-plugin-ros2dds/src/ros2_utils.rs

```diff
@@ -228,16 +228,22 @@ impl CddsRequestHeader {
         buf[16] = self.is_little_endian as u8;
 
         hashmap.insert(ATTACHMENT_KEY_REQUEST_HEADER, buf);
```

Comment:
I'm not sure why a hashmap was used... probably for ease of serialization via `ZBytes::from_iter(hashmap.iter())`.
But now that this ease is gone, I don't see the point of this hashmap. We could just do:
```rust
writer.append(ZBytes::from(ATTACHMENT_KEY_REQUEST_HEADER));
writer.append(ZBytes::from(buf));
writer.finish();
```